sparse_strips/vello_hybrid/src/render.rs

```diff
@@ -200,6 +209,10 @@ impl Renderer {
             render_pipeline,
             resources: None,
             alpha_data: Vec::new(),
+            render_size: RenderSize {
+                width: render_target_config.width,
+                height: render_target_config.height,
```

Comment:
I'm not sure how much value this adds - these values never be used (because when they would have been used, the render resources will have been created for the first time anyway). I think it would be just as valid for this to be any values (including probably 0x0). If this analysis is right, it being on the `RenderTargetOptions` is misleading.